aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunClientFactories.java

```diff
@@ -81,17 +88,82 @@ private static AliyunClientFactory loadClientFactory(
   }
 
   static class DefaultAliyunClientFactory implements AliyunClientFactory {
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultAliyunClientFactory.class);
     private AliyunProperties aliyunProperties;
 
     DefaultAliyunClientFactory() {}
 
+    /**
+     * Check if RRSA environment variables are present. RRSA requires
```

Comment:
Is it possible to add more description about the `RRSA` ?  Actually I also don't have much context about this, which I think we need to provide more context for others to understand and mantain this.